tests/test_component.py

```diff
@@ -130,15 +126,15 @@ def _process_dataset(self, manifest: Manifest) -> t.Union[None, dd.DataFrame]:
         "list_default_arg": ["foo", "bar"],
         "dict_default_arg": {"foo": 1, "bar": 2},
         "string_default_arg": "foo",
-        "string_default_arg_none": None,
-        "integer_default_arg_none": None,
-        "float_default_arg_none": None,
-        "bool_default_arg_none": None,
-        "list_default_arg_none": None,
-        "dict_default_arg_none": None,
+        "string_default_arg_none": "None",
```

Comment:
This should be Nontype and not string (see suggestions above). 
Same for the last two arguments